### PR TITLE
Adds the Counselor's Session Submissions FAQ to the site

### PR DESCRIPTION
**Pull request by astralbodies**
Adds the basic FAQ for session submissions to the site. I'm not entirely sure where to link it from so for now this creates the markdown document only.

### CODE REVIEW

File: markdown/counselors-submission-faq.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+We've collected a number of questions that we feel may (or have already been) frequently asked surrounding the process of submitting a session to speak at THAT Conference. If your question isn't answered here feel free to reach out in Slack in the #sessions channel or via e-mail at [Hello@THATConference.com](mailto:Hello@THATConference).
+
+### Why do I need a new account?
+
+After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including user accounts. You have to create a new account on the site.
+
+### What happened to my old sessions?
+
+All of the old session data has been deleted as the site is completely new this year. We may decide to migrate the old data over to the new site and connect it with your profile. Things are still under development so we're not 100% sure what will happen.
```
Comment (brettski) [inline markdown/counselors-submission-faq.md]: Have this follow the same as account:
`After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including sessions.`
The continue with "We may decide to migrate..."

File: markdown/counselors-submission-faq.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### My child is under 13 years of age, what should I do?
+
+We don't support accounts for users under the age of 13 mostly to be in compliance with State and Federal laws around child safety online. If your child would like to speak at THAT Conference, please submit the session under your own account. There is a field on the session form that lets you tell us anything else that's important for us to know - tell us there that you're submitting on behalf of your child. If the session is accepted, we'll work with you to get their information for the site.
```
Comment (brettski) [inline markdown/counselors-submission-faq.md]: Suggest drop "mostly" from the paragraph

File: markdown/counselors-submission-faq.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+### Why can I only submit three sessions for consideration?
+
+Each session is reviewed by an individual of our volunteer staff and also voted on by our community. We asked that you limit your submissions to the three best ideas that you have to help us pick the best possible sessions overall.
```
Comment (brettski) [inline markdown/counselors-submission-faq.md]: drop "volunteer"

File: markdown/counselors-submission-faq.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+### Why can I only submit three sessions for consideration?
+
+Each session is reviewed by an individual of our volunteer staff and also voted on by our community. We asked that you limit your submissions to the three best ideas that you have to help us pick the best possible sessions overall. 
+
+### Can I get help with developing my session submission?
+
+Sure! Firstly, we suggest you join our community Slack and check out the #sessions channel. Conference staff and other previous counselors are there to help you out! 
+
+### Can I have dinner with Clark?
```
Comment (brettski) [inline markdown/counselors-submission-faq.md]: Funny, but should be removed

**Comment by astralbodies** [comment]
@brettski Adjusted the copy per your review and rename the file.

**Pull request merged**